docs/strat-lib/getting-started/smart-offer.md

```diff
@@ -270,28 +204,36 @@ cast send --rpc-url $LOCAL_URL "$WBTC" "mint(uint)" 500000000 --private-key "$PR
 If the admin acts as taker and takes the offer (see [snipe guide](../guides/howToSnipe.md)), you will also need USDT.
 
 ```bash
-cast send --rpc-url $LOCAL_URL "$USDT" "mint(uint)" 100000000000 --private-key "$PRIVATE_KEY"
+cast send --rpc-url $LOCAL_URL "$USDT" "mint(uint)" 1000000000000 --private-key "$PRIVATE_KEY"
 ```
 
 #### Approving the contract to pull the funds
 
-The router needs to be able to pull funds from the admin. First we need to get the address of the router:
-
 ```bash
-cast call --rpc-url $LOCAL_URL "$OFFER_MAKER" "router()(address)"
+cast send --rpc-url $LOCAL_URL "$WBTC" "approve(address, uint)" "$OFFER_MAKER" 100000000 --private-key "$PRIVATE_KEY"
```

Comment:
this will not have an effect. The OFFER_MAKER does not have functionality to pull funds from the user. The offer will fail unless you transfer funds to the contract since we do not have a router.

Reply:
yes